frontend/projects/valtimo/case-management/src/lib/components/case-management-detail/tabs/case-management-actions/case-management-actions-modal/case-management-actions-modal.component.scss

```
@@ -0,0 +1,64 @@
+
+  &:hover:not(:disabled) {
+    background-color: var(--cds-background-inverse-hover, #474747);
+  }
+
+  &:disabled {
+    opacity: 0.5;
+    cursor: not-allowed;
+  }
+
+  &__icon {
+    fill: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)